sources/procfs.go

```diff
@@ -320,6 +325,14 @@ func (s *lustreProcfsSource) generateClientMetricTemplates() error {
 			{"stats", "inode_permission_total", inodePermissionHelp, s.counterMetric},
 			{"xattr_cache", "xattr_cache_enabled", "Returns '1' if extended attribute cache is enabled", s.gaugeMetric},
 		},
+		"mdc/*": {
+			{"rpc_stats", "rpc_in_flight", rpcsInFlightHelp, s.gaugeMetric},
```

Comment:
I see. You are right, we would need to prepend the values with the location. The labels would be identical since they are both on the same node and the `nodeType` will always be `client` for these two. Also, the metrics can and likely always will be populated at the same time, so we definitely need to change them.